### DISCUSSION TITLE
q-drawer calls update:modelValue after click on expanded list item on mobile view

### DISCUSSION THREAD

**Discussion by tarikrital** [quasarframework/quasar#13787] [General - Components / Directives / etc] [answered]
The desktop mode works ok and sidebar shows with expanded list working but on `mobile` view after clicking the blue icon on the right to show the **sidebar** it displays but after clicking the item on the left it becomes hidden!

![image](https://user-images.githubusercontent.com/52907282/175788250-5260d80d-fa9d-41fd-969f-9ae92a0c8b20.png)

**Comment by MilosPaunovic** [comment]
Please provide a Codepen reproduction.

**Reply by tarikrital** [reply]
That is why I added a discussion not a bug report which obligates your request. I don't think that is necessary but in short it is like the following:

```
<q-drawer no-swipe-open no-swipe-close no-swipe-backdrop
       behavior="desktop" persistent  v-model="showSidebar" :width="390" side="left"
       :class="isDesktop ? '' : (isPortrait? 'tw-mt-[232px]' : 'tw-mt-[142px]') ">
```

 ```
 setup() {
    const showSidebar  = ref(true)
```

```
  onBeforeMount( async () => {
    showSidebar.value =  $q.screen.lt.md ? false : true
    })
```

**Comment by tarikrital** [comment, marked as answer]
After setting the `:breakpoint="500"` it works on mobile.
